frontend/jupyter/src/app/pages/index/index-default/index-default.component.ts

```diff
@@ -199,11 +200,14 @@ export class IndexDefaultComponent implements OnInit, OnDestroy {
         if (a.data.status.phase === STATUS_TYPE.TERMINATING) {
           a.event.stopPropagation();
           a.event.preventDefault();
-          this.snackBar.open(
-            'Notebook is being deleted, cannot show details.',
-            SnackType.Info,
-            4000,
-          );
+          const config: SnackBarConfig = {
+            data: {
+              msg: 'Notebook is being deleted, cannot show details.',
+              snackType: SnackType.Info,
+            },
+            duration: 4000,
```

Comment:
Refereeing to the other PR which doesn't have a duration... 

Second thing: Should we have a constant somewhere so ALL snackbars have the same duration? (If idea liked, please create future ticket or the likes)

Reply:
Like i had mentioned in the other pr, we do have a default set to 3000 https://github.com/StatCan/jupyter-apis/pull/338/files#diff-2f7010f58ad720430fd41c3dddc60949ad444212976a7759d5916901d421c02eR64

So it would be very easy to set it to one common value (the default) and then not specify the duration on each snackbar config. 
I dont hate that idea to have all snackbars share the same length. So I will make a ticket about it like you said. We can use that time to do a bit of investigation/testing on if we should have different durations or not.

Reply:
issue created for this https://jirab.statcan.ca/browse/BTIS-855